hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/s3/GetS3SecretHandler.java

```diff
@@ -35,8 +35,8 @@
 public class GetS3SecretHandler extends S3Handler {
 
   @Option(names = "-u",
-      description = "Specify the user name to perform the operation on "
-          + "(admins only)'")
+      description = "Specify the user to perform the operation on "
+          + "(Admins only)'")
```

Comment:
Is there a plan to allow tenant admins to get secret for a tenant user? If the user has lost the secret, only an ozone admin can retrieve it then. Or, is there a plan to add a new 'tenant' based command for getting accessId / secret key for a user?

Reply:
Good question. I'm not sure if we should allow **tenant** admins to get secrets for it's users though. @prashantpogde Maybe we could optionally allow that?

Reply:
The user should be able to retrieve their own secrets on ozone CLI. We can allow either the ozone admin or user him self to get their secret. Tenant admin need not be given this privilege.

Reply:
Had an offline discussion, we are not allowing tenant admins to retrive other users' secrets in the same tenant. Only Ozone admins can do that.